### PR TITLE
NewRelic - Updated configuration definition

### PR DESCRIPTION
**Pull request by indigobravo**
This branch introduces a new method for configuring the newrelic-admin
script launched via supervisord.conf. The three processes launched by
supervisor have been wrapped with their own application names

If this works it will enable us to filter NewRelic for specific results
or each process / component we are running.

This remains untested, and we need to make sure it works correctly
before pushing to production. I suggest deploying to QA and checking the
newrelic interface to see what is being reported.

### CODE REVIEW

File: conf/supervisord.conf [hunk 1]
```diff
@@ -12,21 +12,23 @@ stdout_events_enabled=true
 stderr_events_enabled=true
 
 [program:web]
-environment=NEW_RELIC_CONFIG_FILE=conf/app.ini
+NEW_RELIC_CONFIG_FILE=conf/newrelic-web.ini
```
Comment (jon-betts) [inline conf/supervisord.conf:15]: I'm not sure you can just put environment variables in here like this. I think you need to use `environment=` with key value pairs: http://supervisord.org/configuration.html#supervisord-section-values

Also, as we only want to set one thing, can we get it all done here?
```suggestion
environment=NEW_RELIC_APP_NAME=h-web
```

Comment (indigobravo) [reply conf/supervisord.conf]: Thanks Jon - Updates have been made and pushed.

File: conf/supervisord.conf [hunk 2]
```diff
@@ -12,21 +12,23 @@ stdout_events_enabled=true
 stderr_events_enabled=true
 
 [program:web]
-environment=NEW_RELIC_CONFIG_FILE=conf/app.ini
+environment=NEW_RELIC_APP_NAME=h-web
```
Comment (robertknight) [inline conf/supervisord.conf:15]: We don't have any `[newrelic]` section in the `conf/app.ini` currently so I believe it should be fine to remove the `NEW_RELIC_CONFIG_FILE` here. We might need to re-add it in future if we want to customize the [Python agent configuration](https://docs.newrelic.com/docs/agents/python-agent/configuration/python-agent-configuration#agent-configuration-file).

**Review by robertknight — APPROVED** [review]
LGTM. Let's test it.

File: conf/supervisord.conf (hunk 2 above)
Comment (indigobravo) [reply conf/supervisord.conf:15]: Agreed - A previous suggestion was to have a config file for each component. However, if we are only setting the application name it is a little over the top.

**Pull request merged**